test/find.cpp

```diff
@@ -158,17 +158,18 @@ TEST(FindTests, ByPath)
     ASSERT_EQ(range5.begin()->getIndex(), 0);
```

Comment:
This assertion has now become superfluous.

Reply:
Yes, this should be removed.